layouts/partials/head.html

```diff
@@ -53,6 +53,7 @@
                 </div>
                 <div id="navbar" class="collapse navbar-collapse">
                     <ul class="nav navbar-nav">
+                        <li><a href="/blog/">Blog</a></li>
```

Comment:
Needs a repeat in `layouts/partials/home-head.html` for the front page, if that was your intent?

Reply:
Nope, I missed that, thanks!

Reply:
Ah, wait, I intended that. I want to redo the home page in a seperate PR